public-ui/src/app/views/ehr-view/ehr-view.component.html

```diff
@@ -1,245 +1,264 @@
-          <div class="col-xs-12 db-alert">
-            <clr-alert  [clrAlertClosable]="false" [clrAlertType]="info">
+        <div *ngIf="showTopConcepts">
+          <app-chart *ngIf="showTopConcepts" class="top-concepts-chart" [concepts]="top10Results" [pointWidth]="10">
+          </app-chart>
+        </div>
+      </section>
+      <div class="results-grid" *ngIf="items.length > 0">
+        <div class="domain-info-container">
+          <h5 id="domain-name" class="primary-display">
+            Showing top {{items.length}}&nbsp;
+          </h5>
+          <h5 id="domain-name" class="primary-display" *ngIf="searchText.value.length === 0; else no_search_term">
+            concepts for this domain
+            <clr-tooltip>
+              <clr-icon clrTooltipTrigger shape="info-standard" class="is-solid info-icon"></clr-icon>
+              <clr-tooltip-content clrPosition=top-right clrSize="lg" *clrIfOpen>
+                <div class="tooltip-label">{{matchingConceptsHelptText}}</div>
+              </clr-tooltip-content>
+            </clr-tooltip>
+          </h5>
+          <ng-template #no_search_term>
+            <h5 id="domain-name" class="primary-display" *ngIf="searchText.value.length > 0">
+              matching medical concepts
+              <clr-tooltip>
+                <clr-icon clrTooltipTrigger shape="info-standard" class="is-solid info-icon"></clr-icon>
+                <clr-tooltip-content clrPosition=top-right clrSize="lg" *clrIfOpen>
+                  <div class="tooltip-label">{{matchingConceptsHelptText}}</div>
+                </clr-tooltip-content>
+              </clr-tooltip>
+            </h5>
+          </ng-template>
+        </div>
+        <div *ngIf="items.length === 1 && items[0].standardConcept != 'S' && standardConcepts.length > 0">
+          <div class=" db-alert">
+            <clr-alert [clrAlertClosable]="false" [clrAlertType]="info">
               Note: {{items[0].vocabularyId}} {{items[0].conceptCode}} "{{items[0].conceptName}}"
               maps to Standard Vocabulary {{standardConcepts[0].vocabularyId}} {{standardConcepts[0].conceptCode}}.
               Standard vocabularies capture data across a variety of source vocabularies.
             </clr-alert>
           </div>
         </div>
-        <div class="row">
-          <div class="col-md grid-header flex-xs-middle"> {{ehrDomain.name}}
+
+        <div class="tbl-r tbl-head">
+          <div class="tbl-d first secondary-display info-text"> {{ehrDomain.name}}
             <clr-tooltip>
-              <clr-icon clrTooltipTrigger shape="info-standard" class="info-icon"></clr-icon>
-              <clr-tooltip-content clrPosition=right
-                                   clrSize="lg" *clrIfOpen>
+              <clr-icon clrTooltipTrigger shape="info-standard" class="is-solid info-icon"></clr-icon>
+              <clr-tooltip-content clrPosition=top-right clrSize="lg" *clrIfOpen>
```

Comment:
If I try to open the tooltip here it gives me below errors.
<img width="1519" alt="screen shot 2019-02-27 at 4 12 57 pm" src="https://user-images.githubusercontent.com/42041705/53526696-90d15500-3aaa-11e9-90d4-0abd09c3d902.png">